### PR TITLE
Feature/nvimtree configuration

### PR DESCRIPTION
**Pull request by krsntn**
allow user to configure nvimtree setting in lv-config.lua file

### CODE REVIEW

File: utils/installer/lv-config.example.lua [hunk 1]
```diff
@@ -28,6 +28,12 @@ O.plugin.indent_line.active = false
 O.plugin.zen.active = false
 O.plugin.zen.window.height = 0.90
 
+-- nvimtree
+O.nvimtree.side = "left"
```
Comment (abzcoding) [inline utils/installer/lv-config.example.lua:32]: I think it is better to comment these lines out. to show that you can change them
cause that would [change](https://github.com/ChristianChiarulli/LunarVim/blob/rolling/utils/installer/install.sh#L115) the behavior for new installs.

Comment (krsntn) [reply utils/installer/lv-config.example.lua]: Cool! got it. I have made new commits to comment it out.

**Review by abzcoding — APPROVED** [review]
thank you for your contribution

**Comment by abzcoding** [comment]
nvim-tree is already user-configurable through `lua/core/nvimtree.lua`->`M.config` function

```lua
-- lv-config.lua

O.plugin.nvimtree.side = "right"
-- or any other config you want under O.plugin.nvimtree.
```

tree

**Pull request closed without merging**